spec/models/population_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe Population do
+  it { is_expected.to belong_to(:location) }
+
+  it { is_expected.to validate_presence_of(:date) }
+  it { is_expected.to validate_presence_of(:operational_capacity) }
+  it { is_expected.to validate_presence_of(:usable_capacity) }
+  it { is_expected.to validate_presence_of(:unlock) }
+  it { is_expected.to validate_presence_of(:bedwatch) }
+  it { is_expected.to validate_presence_of(:overnights_in) }
+  it { is_expected.to validate_presence_of(:overnights_out) }
+  it { is_expected.to validate_presence_of(:out_of_area_courts) }
+  it { is_expected.to validate_presence_of(:discharges) }
+  it { is_expected.to validate_numericality_of(:operational_capacity) }
```

Comment:
Can we test the minimum restriction in these validations?

Reply:
Yes, I'll add that